### PR TITLE
Add back some sections removed from gitea repo README

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: content/page/index.en-us.md [hunk 1]
```diff
@@ -15,6 +15,35 @@ Gitea is a painless self-hosted Git service. You can compare it with GitHub, Bit
...
 The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across ALL platforms that Go supports, including Linux, macOS and Windows, even on architectures like ARM or PowerPC.
 
+## Features
+
+- Activity timeline
+- SSH and HTTP/HTTPS protocols
+- SMTP/LDAP/Reverse proxy authentication
+- Reverse proxy with sub-path
+- Account/Organization/Repository management
+- Add/Remove repository collaborators
```
Comment (bkcsoft) [inline content/page/index.en-us.md:25]: Just `- Repository Collaboration` ?

**Comment by bkcsoft** [comment]
Otherwise LGTM

File: content/page/index.en-us.md [hunk 2]
```diff
@@ -15,6 +15,35 @@ Gitea is a painless self-hosted Git service. You can compare it with GitHub, Bit
...
+- Repository/Organization webhooks (including Slack)
+- Repository Git hooks/deploy keys
+- Repository issues, pull requests and wiki
+- Migrate and mirror repository and its wiki
+- Web editor for repository files and wiki
+- Gravatar and Federated avatar with custom source
+- Mail service
+- Administration panel
+- Supports MySQL, PostgreSQL, SQLite3 and [TiDB](https://github.com/pingcap/tidb) (experimental)
```
Comment (tboerger) [inline content/page/index.en-us.md:34]: Mssql :)

File: content/page/index.en-us.md [hunk 3]
```diff
@@ -15,6 +15,35 @@ Gitea is a painless self-hosted Git service. You can compare it with GitHub, Bit
...
+- Multi-language support ([21 languages](https://github.com/go-gitea/gitea/tree/master/options/locale))
```
Comment (tboerger) [inline content/page/index.en-us.md:35]: Was it 22?

File: content/page/index.en-us.md [hunk 4]
```diff
@@ -35,3 +64,18 @@ The goal of this project is to make the easiest, fastest, and most painless way
   * [github.com/lib/pq](https://github.com/lib/pq)
   * [github.com/mattn/go-sqlite3](https://github.com/mattn/go-sqlite3)
   * [github.com/pingcap/tidb](https://github.com/pingcap/tidb)
+
+## Software and Service Support
```
Comment (tboerger) [inline content/page/index.en-us.md:68]: Does it really work with all of that?

File: content/page/index.en-us.md [hunk 5]
```diff
@@ -35,3 +64,18 @@ The goal of this project is to make the easiest, fastest, and most painless way
...
+- [Taiga](https://taiga.io/) (Project Management)
+- [Puppet](https://forge.puppetlabs.com/Siteminds/gogs) (IT)
+- [Kanboard](http://kanboard.net/plugin/gogs-webhook) (Project Management)
+- [BearyChat](https://bearychat.com/) (Team Communication)
+- [HiWork](http://www.hiwork.cc/) (Team Communication)
+
+### Product Support
+
+- [Synology](https://www.synology.com) (Docker)
+- [One Space](http://www.onespace.cc) (App Store)
```
Comment (tboerger) [inline content/page/index.en-us.md]: For sure only Gogs and not gitea

File: content/page/index.en-us.md (hunk 3 above)
Comment (lunny) [reply content/page/index.en-us.md:35]: There is a new language we hasn't cherry-pick. I have counting the locale/*, it's 21.

File: content/page/index.en-us.md (hunk 2 above)
Comment (lunny) [reply content/page/index.en-us.md:34]: MSSQL should be v1.1

File: content/page/index.en-us.md (hunk 4 above)
Comment (lunny) [reply content/page/index.en-us.md:68]: Don't know. Just copy from Gogs README

File: content/page/index.en-us.md (hunk 5 above)
Comment (lunny) [reply content/page/index.en-us.md]: I will remove these.

File: content/page/index.en-us.md (hunk 1 above)
Comment (lunny) [reply content/page/index.en-us.md:25]: Why?

Comment (tboerger) [reply content/page/index.en-us.md:25]: Sounds better :), with lowercase c

File: content/page/index.en-us.md [hunk 6]
```diff
@@ -35,3 +64,18 @@ The goal of this project is to make the easiest, fastest, and most painless way
   * [github.com/lib/pq](https://github.com/lib/pq)
   * [github.com/mattn/go-sqlite3](https://github.com/mattn/go-sqlite3)
   * [github.com/pingcap/tidb](https://github.com/pingcap/tidb)
+
+## Software and Service Support
+
+- [Drone](https://github.com/drone/drone) (CI)
+- [Fabric8](http://fabric8.io/) (DevOps)
+- [Taiga](https://taiga.io/) (Project Management)
+- [Puppet](https://forge.puppetlabs.com/Siteminds/gogs) (IT)
```
Comment (tboerger) [inline content/page/index.en-us.md]: This only installs Gogs via puppet.

File: content/page/index.en-us.md [hunk 7]
```diff
@@ -35,3 +64,18 @@ The goal of this project is to make the easiest, fastest, and most painless way
...
+- [Kanboard](http://kanboard.net/plugin/gogs-webhook) (Project Management)
```
Comment (tboerger) [inline content/page/index.en-us.md]: A Gogs plugin, but maybe still works with gitea.

Comment (lunny) [reply content/page/index.en-us.md]: So let's move them all at first and add one by one.

**Comment by tboerger** [comment]
LGTM

**Pull request merged**